openpdf/src/main/java/com/lowagie/text/Element.java

```diff
@@ -321,29 +324,30 @@ public interface Element {
      * @return    true if this is a 'content' element; false if this is a 'metadata' element
      */
 
-    boolean isContent();
+    public boolean isContent();
     
```

Comment:
Java interface is public by default, NO need to declare it as public, which makes code heavy.

Reply:
sure